docs/source/models.rst

```diff
@@ -2000,6 +2000,15 @@ and standardizes both the inputs and outputs of pipeline inference. This conform
 and batch inference by coercing the data structures that are required for ``transformers`` inference pipelines
 to formats that are compatible with json serialization and casting to Pandas DataFrames.
 
+.. note::
+    Certain `TextGenerationPipeline` types, particularly instructional-based ones, may return the original
+    prompt and included line-formatting carriage returns `"\n"` in their outputs. For these pipeline types,
+    if you would like to disable the prompt return, you can set the following in the `inference_config` dictionary when
+    saving or logging the model: `"include_prompt": False`. To remove the newline characters from within the body
+    of the generated text output, you can add the `"remove_newlines": True` option to the `inference_config` dictionary.
```

Comment:
How about the option name like `shrink_newlines`?

Reply:
Other candidates (rejected in me):

- `replace_newlines`: makes me wonder "replace with what?"
- `replace_newlines_with_space`: clear but too long

Reply:
I like it :) changing!